packages/example/src/components/SetupLightning/styles.ts

```diff
@@ -1,8 +1,7 @@
-import styled from "styled-components";
-import Confetti from "react-confetti";
-import { Modal } from "semantic-ui-react"
-import { Button as SnapButton } from "snapkit"
-
+import styled, { css } from 'styled-components';
```

Comment:
too big to be displayed in github, time to extract styles to separate folders along with component.